chapter-1-basics.md

```diff
@@ -44,9 +44,13 @@ only the `Assembling` step should be covered by MORYX. These information result
 in the following command:
 
 ```
-$ moryx new PencilFactory --steps Assembling --products GraphitePencil
+$ moryx new PencilFactory --steps Assembling --products GraphitePencil --branch adp
 ```
 
+> NOTE:
+> For the application developer program (ADP) a custom template is used.
+> This removes some autogenerated code for simplicity.
```

Comment:
I'd prefer sth like this: "For this ... a simplified template is used, which is tailored to the training scenario"

I think you had that included before, and now that I think about it, you might want to add the information about that `--branch adp` parameter to give some more context. Otherwise this note looks a bit out of nowhere. Something, that covers the following would be good: "In a real world context/application you would either leave the `--branch` parameter or use it maybe in combination with `--template` (didn't check these for correctness) to generate from a custom template that fits your needs"